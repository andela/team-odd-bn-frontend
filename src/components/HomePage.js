import React, { Component } from 'react';
import '../assets/css/HomePage.css';
import destination from '../assets/images/destination.jpeg';

export class HomePage extends Component {
  constructor(props) {
    super(props);
    this.decode = '';
    this.state = {};
  }

  render() {
    return (
      <div className="home">
        <h3 className="home-title">Home</h3>
        <div className="row">
          <div className="column-top">
            <p className="destination-name">Most Travelled Destination</p>
            <div className="card-top">
              <img src={destination} alt="most travelled" />
              <div className="destination-details">
                <p className="destination">Kigali</p>
                <p>Visited 3 times </p>
                <p>4 Accommodations</p>
              </div>
            </div>
          </div>
        </div>

        <div className="row-bottom">
          <div className="column-two">
            <div className="card-bookings">
              <h3 className="request-title">Your Bookings</h3>
              <p className="requests">You have 5 bookings</p>
              <a href="#" className="new-requests">New Booking</a>
            </div>
          </div>

          <div className="column-three">
            <div className="card-requests">
              <h3 className="request-title">Your Requests</h3>
              <p className="requests">You have 5 trip requests</p>
              <a href="#" className="new-requests">New Request</a>
            </div>
          </div>
        </div>
      </div>

    );
  }
}


export default HomePage;
