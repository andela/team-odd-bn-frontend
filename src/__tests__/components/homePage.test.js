import React from 'react';
import { shallow } from 'enzyme';
import HomePage from '../../components/HomePage';

describe('Test Dashboard Homepage page ', () => {
  it('should render', () => {
    const wrapper = shallow(<HomePage />);
    expect(shallow(<HomePage />)).toMatchSnapshot();
  });
});
