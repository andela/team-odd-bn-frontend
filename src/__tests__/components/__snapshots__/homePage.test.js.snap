// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Dashboard Homepage page  should render 1`] = `ShallowWrapper {}`;
